### PR TITLE
Update index.tsx

### PR DESCRIPTION
**Pull request by ninosamson**

### CODE REVIEW

**Review by arranfw — APPROVED** [review]

File: pages/index.tsx [hunk 1]
```diff
@@ -71,13 +71,13 @@ export default function Home() {
           <li>
             <div>
               Individuals who are unvaccinated or have not received their third booster dose{' '}
-              <span className='font-bold'>and</span>
+              <span className='font-bold'>and any of</span>
```
Comment (arranfw) [inline pages/index.tsx]: > third booster dose and any of -> Have three or more chronic

I think this is how it reads, is that the intention?

**Review by arranfw — APPROVED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (arranfw) [reply pages/index.tsx]: image

Comment (ninosamson) [reply pages/index.tsx]: hmm - good call. that's what they said to update, but I'll double check.

Comment (ninosamson) [reply pages/index.tsx]: updated and requesting your re-approval.

**Review by arranfw — APPROVED** [review]

**Pull request merged**